### PR TITLE
Clustering profile

### PR DESCRIPTION
**Pull request by lucasponce**
@jshaughn finally the clustering environment and test could be automated.
This PR creates a domain with:
- 1 node for Cassandra
- 3 nodes forming a hawkular-alerts cluster
that is started via maven plugin and specific clustering tests are running against it.
Hope this one will help to maintain clustering configuration.
For the moment, I won't include it on travis, as there is not enough capacity in the VM .

### CODE REVIEW

File: hawkular-alerts-rest-tests/pom.xml [hunk 1]
```diff
@@ -219,7 +219,6 @@
               <excludes>
                 <excludeFile>org/hawkular/alerts/rest/ClusterITest.class</excludeFile>
                 <excludeFile>org/hawkular/alerts/rest/PerfCrudITest.class</excludeFile>
-                <exclude>${hawkular.itest-exclude}</exclude>
```
Comment (jshaughn) [inline hawkular-alerts-rest-tests/pom.xml]: Why did you remove this?

Comment (lucasponce) [inline hawkular-alerts-rest-tests/pom.xml]: I think it was showing an error in the editor and I don't remember if it was used in this profile.
I can bring it back if it was a mistake.

File: hawkular-alerts-rest-tests/src/test/resources/host.xsl [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+  </xsl:template>
+
+  <!-- copy everything else as-is -->
+  <xsl:template match="node()|@*">
+    <xsl:copy>
+      <xsl:apply-templates select="node()|@*" />
+    </xsl:copy>
+  </xsl:template>
+
+</xsl:stylesheet>
```
Comment (jshaughn) [inline hawkular-alerts-rest-tests/src/test/resources/host.xsl]: newline?

Comment (lucasponce) [inline hawkular-alerts-rest-tests/src/test/resources/host.xsl]: sure :)

File: hawkular-alerts-rest-tests/pom.xml (hunk 1 above)
Comment (jshaughn) [inline hawkular-alerts-rest-tests/pom.xml]: I use it sometimes for additional exclusions.  It was just a convenience.

Comment (lucasponce) [inline hawkular-alerts-rest-tests/pom.xml]: No problem. I will restore it.
I thought it was old code left there.
